packages/server/src/hono/routes/a2a-tasks.ts

```
@@ -0,0 +1,365 @@
+                name: z.string().optional(),
+                mimeType: z.string().optional(),
+            }),
+        ]),
+        metadata: z.record(z.any()).optional(),
+    }),
+    z.object({
+        kind: z.literal('data'),
+        data: z.record(z.any()),
+        metadata: z.record(z.any()).optional(),
+    }),
+]);
```

ESLint

[error] Zod method chain is missing .describe() call. Add .describe("description here") for OpenAPI documentation.

(dexto-custom/require-zod-describe)